firestore/user_reports.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
+
+import dateparser
+from cachetools import cached
+from fireo import models as mdl
+
+from firestore.cachemanager import INCIDENT_CACHE, INCIDENT_STATS_CACHE, flush_cache
+
+
+class UserReport(mdl.Model):
+    user_report_time = mdl.DateTime(required=True)
+    created_on = mdl.DateTime(auto=True)
+    user_report_location = mdl.TextField()
+    description = mdl.TextField(required=True)
+    description_translate = mdl.MapField(required=False)
+    attachments = mdl.MapField(required=False)
```

Comment:
whi attachement is MapField?

Reply:
Resolved in https://github.com/1thing-org/hatecrimetracker/pull/16